packages/bot-skeleton/src/scratch/dbot.js

```diff
@@ -24,6 +26,67 @@ class DBot {
     async initWorkspace(public_path, store, api_helpers_store, is_mobile) {
         const recent_files = await getSavedWorkspaces();
 
+        api_base.init();
+        this.interpreter = Interpreter();
+        const that = this;
+        Blockly.Blocks.trade_definition_tradetype.onchange = function (event) {
```

Comment:
@vinu-deriv Bhai is it somehow possible to keep this onChange in the previous [trade_definition_tradetype.js] file where it was or it will miss some references in that way?